pages/index/index.js

```diff
@@ -3,6 +3,7 @@ import glamorous from 'glamorous'
 import {LiveProvider, LiveEditor, LiveError} from 'react-live'
 import {LiveContextTypes} from 'react-live/lib/components/Live/LiveProvider'
 import Head from 'next/head'
+import {AlgoliaLink, AlgoliaScript} from '../../components/algolia-config'
```

Comment:
After looking in the docs and not seeing anything, I tried again and it seems all fine with multiple head components. have updated :)